source/blog.liquid

```diff
@@ -0,0 +1,32 @@
+---
+pagination:
+  data: collections.blog
+  size: 5
+  alias: posts
+  reverse: true
+layout: has_no_sidebars
+title: Sass Blog
+---
+
+Page {{ pagination.pageNumber | plus: 1 }} of {{ pagination.pages.length }}
+<ul class="sl-c-list">
+  {% if pagination.href.previous %}
+    <li>
+      <a href="{{ pagination.href.previous }}">Previous</a>
+    </li>
+  {% endif %}
+  {%- for post in posts -%}
+    <li>
+      <h2>
+        <a href="{{post.url}}">{{ post.data.title }}</a>
+      </h2>
+      <p>Posted {{ post.date | formatBlogDate }} by {{ post.data.author }}</p>
+      <p>{{ post.content | truncatePost }}</p>
+    </li>
+  {%- endfor -%}
+  {% if pagination.href.next %}
+    <li>
+      <a href="{{ pagination.href.next }}">Next</a>
```

Comment:
The live site has "Next Page". Not sure if we wanted to stick with that or if we made this change intentionally.

Reply:
I noticed that the IDs are no longer included on the headings (that aren't the post title itself). I am not positive, but I think we spoke about trying to get the same IDs on the headings that the live site has in order to not break a link that may be pointed to the site/#anchor. I don't know if that would apply here, but I think we were discussing it in regards to the work Sana is doing on the Learn Sass page.

Reply:
Which IDs? If you mean the page anchors we moved that to separate ticket @stacyk

Reply:
@oluoluoxenfree I'm in this code doing some review, so I can add this in 👍

Reply:
https://trello.com/c/T0UW4E0y